downstream/modules/devtools/ref-rhdh-install-ocp-prereqs.adoc

```diff
@@ -0,0 +1,11 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="rhdh-install-ocp-prereqs_{context}"]
+= Prerequisites
+
+* {RHDH} installed on {OCP} using a Helm chart.
+Follow the steps in the link:{BaseURL}/red_hat_developer_hub/1.2/html-single/getting_started_with_red_hat_developer_hub/proc-install-rhdh-helm_rhdh-getting-started[Deploying Red Hat Developer Hub on OpenShift Container Platform] chapter of _Administration Guide for Red Hat Developer Hub_.
```

Comment:
The destination of this link doesn't match with either of the titles which can be confusing for the user. The correct title is used in your google doc ("Getting started with Red Hat Developer Hub")